src/main/clojure/aerospike_clj/mock_client.clj

```diff
@@ -1,12 +1,25 @@
 (ns aerospike-clj.mock-client
   (:refer-clojure :exclude [update])
   (:require [promesa.core :as p]
+            [aerospike-clj.client :as client]
             [aerospike-clj.protocols :as pt]
             [aerospike-clj.utils])
-  (:import (com.aerospike.client AerospikeException ResultCode)))
+  (:import (com.aerospike.client AerospikeException ResultCode)
+           (java.time Instant)))
 
 (def ^:private DEFAULT_SET "__DEFAULT__")
 
+(defn now-epoch
+  "Mocks `now` in unix epoch for TTL usages.
+  Returns the epoch time of 01/01/2022. Can be redefined for using a different `now`"
+  []
+  (.getEpochSecond (Instant/parse "2022-01-01T00:00:00.00Z")))
+
+(defn expiration->ttl
+  "converts an expiration into TTL as seconds from `Aerospike epoch`"
+  [expiration]
+  (- (+ (now-epoch) expiration) client/EPOCH))
```

Comment:
I'm not sure I understand why we need `now-epoch` to convert an expiry to an absolute TTL from `now`.

Reply:
For tests to have deterministic output. Using the real now means each run returns a different result.

Reply:
I've re-implemented this as a fixed clock to make its meaning clearer.